crates/wash-lib/src/wait.rs

```diff
@@ -126,7 +126,7 @@ pub async fn wait_for_component_scaled_event(
         }
 
         match cloud_event.event_type.as_str() {
-            "com.wasmcloud.lattice.component_scaled" | "com.wasmcloud.lattice.actor_scaled" => {
+            "com.wasmcloud.lattice.component_scaled" | "com.wasmcloud.lattice.actor_scaled" => { // NOTE(nitame): Is actor still need to be part of match case?
```

Comment:
If actor_scaled and actor_scale_failed do not need to be matched any more. I can remove them and the handling of both actor_id and component_id.

Reply:
@brooksmtownsend I think we can drop the actor_scale* events now, right? I'm not remembering us using them anywhere anymore

Reply:
Yep! We can remove `actor_scaled`. Nothing wrong with it being in there but it'll be nice to get rid of the extra case + handling of actor_id